### PR TITLE
Implement expand/collapse and update ug

### LINKED ISSUE #84: Folder enhancement
- [ ] expand collapse
- [ ] edit folder name
- [ ] delete folder

### PR DESCRIPTION
**Pull request by openorclose**
No tests for the commands because unable to test ui, so just do some manual testing.

Resolves #84 (except folder delete)

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -679,13 +679,9 @@ Expands the folder hierarchy by the specified number of levels.
 Format: `*expand* [LEVEL]`
```
Comment (Na-Nazhou) [inline docs/UserGuide.adoc]:
```suggestion
Format: `*expand* [LEVEL=1]`
```

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -697,16 +693,12 @@ Expands the folder hierarchy by 3 levels.
 
 Collapses the folder hierarchy by the specified number of levels.
 
-Format: `*collapse* [LEVEL]`
+Format: `*collapse* [LEVEL = 1]`
```
Comment (Na-Nazhou) [inline docs/UserGuide.adoc]:
```suggestion
Format: `*collapse* [LEVEL=1]`
```

File: src/main/java/seedu/mark/logic/commands/CollapseCommand.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    public static final String COMMAND_WORD = "collapse";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Collapses the folder hierarchy.\n"
+            + "If not specified, the default level to collapse by is 1.\n"
+            + "Parameters: [LEVELS] (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_GOTO_BOOKMARK_ACKNOWLEDGEMENT = "Collapsing folders by %1$d level(s)";
```
Comment (Na-Nazhou) [inline src/main/java/seedu/mark/logic/commands/CollapseCommand.java]:
```suggestion
    public static final String MESSAGE_COLLAPSE_FOLDER_ACKNOWLEDGEMENT = "Collapsing folders by %1$d level(s)";
```

File: src/main/java/seedu/mark/logic/commands/CollapseCommand.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    public CollapseCommand(int levelsToCollapse) {
+        this.levelsToCollapse = levelsToCollapse;
+    }
+
+    @Override
+    public CommandResult execute(Model model, Storage storage) {
+        requireAllNonNull(model, storage);
+
+        return new ExpandCommandResult(
+                String.format(MESSAGE_GOTO_BOOKMARK_ACKNOWLEDGEMENT, levelsToCollapse), -levelsToCollapse);
```
Comment (Na-Nazhou) [inline src/main/java/seedu/mark/logic/commands/CollapseCommand.java]:
```suggestion
                String.format(MESSAGE_COLLAPSE_FOLDER_ACKNOWLEDGEMENT, levelsToCollapse), -levelsToCollapse);
```

File: src/main/java/seedu/mark/logic/commands/ExpandCommand.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+    public static final String COMMAND_WORD = "expand";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Expands the folder hierarchy.\n"
+            + "If not specified, the default level to expand by is 1.\n"
+            + "Parameters: [LEVEL] (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_GOTO_BOOKMARK_ACKNOWLEDGEMENT = "Expanding folders by %1$d level(s)";
```
Comment (Na-Nazhou) [inline src/main/java/seedu/mark/logic/commands/ExpandCommand.java]:
```suggestion
    public static final String MESSAGE_EXPAND_FOLDER_ACKNOWLEDGEMENT = "Expanding folders by %1$d level(s)";
```

File: src/main/java/seedu/mark/logic/commands/ExpandCommand.java [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    public ExpandCommand(int levelsToExpand) {
+        this.levelsToExpand = levelsToExpand;
+    }
+
+    @Override
+    public CommandResult execute(Model model, Storage storage) {
+        requireAllNonNull(model, storage);
+
+        return new ExpandCommandResult(
+                String.format(MESSAGE_GOTO_BOOKMARK_ACKNOWLEDGEMENT, levelsToExpand), levelsToExpand);
```
Comment (Na-Nazhou) [inline src/main/java/seedu/mark/logic/commands/ExpandCommand.java]:
```suggestion
                String.format(MESSAGE_EXPAND_FOLDER_ACKNOWLEDGEMENT, levelsToExpand), levelsToExpand);
```

File: src/main/java/seedu/mark/logic/commands/ExpandCommand.java [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+ * Expands the folders by a certain amount.
+ */
+public class ExpandCommand extends Command {
+
+    public static final String COMMAND_WORD = "expand";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Expands the folder hierarchy.\n"
+            + "If not specified, the default level to expand by is 1.\n"
+            + "Parameters: [LEVEL] (must be a positive integer)\n"
```
Comment (Na-Nazhou) [inline src/main/java/seedu/mark/logic/commands/ExpandCommand.java]:
```suggestion
            + "Parameters: [LEVELS] (must be a positive integer)\n"
```

Comment (openorclose) [reply src/main/java/seedu/mark/logic/commands/ExpandCommand.java]: I used `LEVEL` everywhere so i think its ok

**Pull request merged**